test/test_tokenizer.rb

```diff
@@ -84,6 +84,24 @@ def test_comments
     assert_equal %w(foo COMMENT""" bar), tokenize("foo\n\"\"\"\nComment\n\"\"\"\nbar")
     assert_equal %w(foo COMMENT''' bar), tokenize("foo\n'''\nComment\n'''\nbar")
 
+    assert_equal %w(foo COMMENT/-), tokenize("foo /- Comment -/")
+    assert_equal %w(foo COMMENT/-), tokenize("foo /-Comment-/")
+    assert_equal %w(foo COMMENT/-), tokenize("foo /- \nComment\n -/")
+    assert_equal %w(foo COMMENT/-), tokenize("foo /-\nComment\n-/")
+    assert_equal %w(foo COMMENT/-), tokenize("foo /-- Comment -/")
+    assert_equal %w(foo COMMENT/-), tokenize("foo /--Comment-/")
+    assert_equal %w(foo COMMENT/-), tokenize("foo /-! Comment -/")
+    assert_equal %w(foo COMMENT/-), tokenize("foo /-!Comment-/")
+    assert_equal %w(foo COMMENT/-), tokenize("foo /-!\nComment\n-/")
+    assert_equal %w(foo COMMENT/- bar), tokenize("foo /-\nComment\n-/ bar")
+    assert_equal %w(foo COMMENT/- bar), tokenize("foo /-\nComment\n-/\nbar")
+    assert_equal %w(foo COMMENT/- bar), tokenize("foo\n/-\nComment\n-/\nbar")
+    assert_equal %w(COMMENT/-), tokenize("/--/")
+    assert_equal %w(COMMENT/-), tokenize("/-\n*\n-/")
+    assert_equal %w(COMMENT/-), tokenize("/-*-/")
+    assert_equal %w(COMMENT/-), tokenize("/-**-/")
+    assert_equal %w(COMMENT/-), tokenize("/-!-/")
+
```

Comment:
Maybe we don't care about it assing, but `not_a_comment /- comment /- also a comment -/ still a comment -/ not_a_comment` would be a good test case